src/main/webapp/maps/placeGuide.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * This class holds a place guide's data, related to its place, creator, image, audio and description.
+ * The place, similarly to the Place class, can be specified either by coordinates or through a place ID.
+ */
+class PlaceGuide {
+  constructor(place, databaseId, description, audioKey, audioLength, imgKey, creatorId, creatorName) {
+    this._place = place;
+    this._databaseId = databaseId;
+    this._description = description;
+    this._audioKey = audioKey;
+    this._audioLength = audioLength;
+    this._imgKey = imgKey;
```

Comment:
Can you change to imageKey to make it the similar with the java class? :)

Reply:
Done.